overlord/configstate/configcore/handlers.go

```diff
@@ -47,6 +47,17 @@ type flags struct {
 	// If set the function should copy such options from values
 	// to early.
 	earlyConfigFilter func(values, early map[string]interface{})
+
+	// preinstallFilter is similar to earlyConfigFilter in that
+	// it allows a handler to express that certain configuration
+	// options should be applied during the preinstall stage before
```

Comment:
again to me is not clear this is just a stage issue because we will point to different kind of partition/root fs, so again the comment needs to be clearer on that

Reply:
see my other response above, but also I did rephrase this comment